ui/redux/reducers/user.js

```diff
@@ -363,6 +363,28 @@ reducers[ACTIONS.USER_PASSWORD_SET_FAILURE] = (state, action) =>
     passwordSetError: action.data.error,
   });
 
+reducers[ACTIONS.ADD_ODYSEE_MEMBERSHIP_DATA] = (state, action) => {
+  return Object.assign({}, state, {
+    odyseeMembershipName: action.data.odyseeMembershipName,
+  });
+};
+
+reducers[ACTIONS.ADD_CLAIMIDS_MEMBERSHIP_DATA] = (state, action) => {
+  let latestData = {};
+
+  // add additional user membership value
+  if (state.odyseeMembershipsPerClaimIds) {
+    latestData = Object.assign(state.odyseeMembershipsPerClaimIds, action.data.response);
```

Comment:
:warning:  
Never mutate `state` in the reducers.   `state.odyseeMembershipsPerClaimIds` is being mutated here.